### ISSUE TITLE
No longer compatible with Django <1.8 since release 0.3.3

### ISSUE THREAD

**Issue by ckesselh** [fabiocaccamo/django-maintenance-mode#12]
Seems the status parameter [got added to the render_to_response method in Django 1.8](http://stackoverflow.com/questions/33212565/render-to-string-got-an-unexpected-keyword-argument-status), so the README.md of django-maintenance-mode will need to be updated.

On Django 1.7, I currently get
```
web      │ ERROR 2016-10-06 19:04:31,614 Internal Server Error: /
web      │ Traceback (most recent call last):
web      │   File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 87, in get_response
web      │     response = middleware_method(request)
web      │   File "/usr/local/lib/python2.7/dist-packages/maintenance_mode/middleware.py", line 57, in process_request
web      │     response = render_to_response(settings.MAINTENANCE_MODE_TEMPLATE, self.get_request_context(request), context_instance=RequestContext(request), content_type='text/html', status=503)
web      │   File "/usr/local/lib/python2.7/dist-packages/django/shortcuts.py", line 25, in render_to_response
web      │     return HttpResponse(loader.render_to_string(*args, **kwargs), **httpresponse_kwargs)
web      │ TypeError: render_to_string() got an unexpected keyword argument 'status'
```

**Comment by fabiocaccamo** [comment]
Fixed with `0.3.4`. Thanks

**Comment by ckesselh** [comment]
I suppose you meant so say "Fixed with 0.4.3". Thanks, mate!

**Comment by fabiocaccamo** [comment]
Yes, sorry :)
